src/sink/prometheus.rs

```diff
@@ -92,40 +117,52 @@ struct PrometheusAggr {
     //     then you probably need the pushgateway (without timestamps) instead.
     //
     // Prometheus has a certain view of the world -- and that's fine -- so we
-    // need to meet it there.
-    inner: Vec<metric::Telemetry>,
-}
-
-fn prometheus_cmp(
-    l: &metric::Telemetry,
-    r: &metric::Telemetry,
-) -> Option<::std::cmp::Ordering> {
-    match l.name.partial_cmp(&r.name) {
-        Some(::std::cmp::Ordering::Equal) => ::metric::tagmap::cmp(&l.tags, &r.tags),
-        other => other,
-    }
+    // need to meet it there. But! We can't just drop metrics as soon as they're
+    // read, apparently, since prometheus expects to be able to re-read multiple
+    // times and get roughly similar results. What we do, then, is store a
+    // sliding window of metrics per hash, keeping only ten values per
+    // timestamp. We sum these values up at report time.
```

Comment:
Is `ten values per timestamp` meant to be a reference to `retain_limit`?

Reply:
Yes. That should be corrected.